releasenotes/notes/update-ci-metadata-extraction-7f5259a18a60fa83.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Only for CI Visibility (``pytest`` integration): Fix calculation of pipeline URL for GitHub Actions.
```

Comment:
Missing new line at EOF 🙁

Reply:
is this enforced by `black`?

Reply:
No, black does not check yaml sources; it's more for keeping the git history as clean as possible 🙂